tests/run_packages_testing.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+http_proxy="${http_proxy:-}"
+https_proxy="${https_proxy:-}"
+DOCKERFILE_PATH="${SCRIPT_PATH}/Dockerfile"
+
+OS_DISTRIBUTION="fedora:31"
+
+install_packages() {
+	for i in "${OS_DISTRIBUTION[@]}"; do
```

Comment:
This is slightly confusing - although you've only defined `OS_DISTRIBUTION` to contain a single distro version, it's actually an array? If so, I'd consider changing the code to be something like:

```sh
OS_DISTRIBUTIONS=()
OS_DISTRIBUTIONS+=("fedora:31")
    :
```